packages/flutter/test/cupertino/scrollbar_test.dart

```diff
@@ -166,6 +167,63 @@ void main() {
     await tester.pump(_kScrollbarFadeDuration);
   });
 
+  testWidgets('Scrollbar changes thickness and radius when dragged', (WidgetTester tester) async {
+    final ScrollController scrollController = ScrollController();
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: MediaQuery(
+          data: const MediaQueryData(),
+          child: PrimaryScrollController(
+            controller: scrollController,
+            child: const CupertinoScrollbar(
+              thickness: 20,
+              thicknessWhileDragging: 40,
+              radius: Radius.circular(10),
+              radiusWhileDragging: Radius.circular(20),
+              child: SingleChildScrollView(child: SizedBox(width: 1600, height: 1200)),
+            ),
+          ),
+        ),
+      ),
+    );
+
+    expect(scrollController.offset, 0.0);
+
+    // Scroll a bit to cause the scrollbar thumb to be shown;
+    // undo the scroll to put the thumb back at the top.
+    const double scrollAmount = 10.0;
+    final TestGesture scrollGesture = await tester.startGesture(tester.getCenter(find.byType(SingleChildScrollView)));
+    await scrollGesture.moveBy(const Offset(0.0, -scrollAmount));
+    await tester.pump();
+    await tester.pump(const Duration(milliseconds: 500));
+    await scrollGesture.moveBy(const Offset(0.0, scrollAmount));
+    await tester.pump();
+    await scrollGesture.up();
+    await tester.pump();
+
+    // Long press on the scrollbar thumb and expect it to grow
+    final TestGesture dragScrollbarGesture = await tester.startGesture(const Offset(780.0, 50.0));
+    await tester.pump(_kLongPressDuration);
+    expect(find.byType(CupertinoScrollbar), paints..rrect(
+      rrect: RRect.fromRectAndRadius(const Rect.fromLTWH(777, 3, 20, 297), const Radius.circular(10)),
```

Comment:
If test is just intended to catch inadvertent changes to the scrollbar's appearance, then `777, 3, 20, 297` etc is OK. If there's simple explanation for the values, it would help to include a comment about that.

Reply:
All magic numbers removed.